src/Kovan-DssSpell.t.sol

```diff
@@ -285,24 +251,24 @@ contract DssSpellTest is DSTest, DSMath {
             dust:         100,             // In whole Dai units
             pct:          400,             // In basis points
             mat:          15000,           // In basis points
-            liqType:      "flip",          // "" or "flip" or "clip"
+            liqType:      "clip",          // "" or "flip" or "clip"
             liqOn:        true,            // If liquidations are enabled
             chop:         1300,            // In basis points
-            cat_dunk:     500,             // In whole Dai units
-            flip_beg:     300,             // In basis points
-            flip_ttl:     1 hours,         // In seconds
-            flip_tau:     1 hours,         // In seconds
-            flipper_mom:  1,               // 1 if circuit breaker enabled
-            dog_hole:     0,
-            clip_buf:     0,
-            clip_tail:    0,
-            clip_cusp:    0,
-            clip_chip:    0,
+            cat_dunk:     0,               // In whole Dai units
+            flip_beg:     0,               // In basis points
+            flip_ttl:     0,               // In seconds
+            flip_tau:     0,               // In seconds
+            flipper_mom:  0,               // 1 if circuit breaker enabled
+            dog_hole:     5 * THOUSAND,
+            clip_buf:     3000,
```

Comment:
We should really fix this way of denoting `buf`--it is perfectly designed to be maximally confusing.

Reply:
Changed!